user/how-to-guides/backup-emergency-restore-v4.md

```diff
@@ -66,44 +66,33 @@ any GNU/Linux system.
 
         [user@restore ~]$ sudo dnf install rpmdevtools
 
- 5. Extract the `scrypt` binary from the RPM.
+ 5. Extract the `scrypt` binary from the RPM and make it conveniently
+    available.
 
         [user@restore ~]$ rpmdev-extract scrypt-*.rpm
+        [user@restore ~]$ alias scrypt="$PWD/scrypt-*/usr/bin/scrypt"
 
 ## Emergency recovery instructions
 
 **Note:** In the following example, the backup file is both *encrypted* and
 *compressed*.
 
- 1. (Optional) If you're working with binaries that you saved with your backup,
-    such as `scrypt`, you can make things easier by aliasing those binaries now,
-    e.g.,
-
-        [user@restore ~]$ alias scrypt="$PWD/scrypt-*/usr/bin/scrypt"
+ 1. Untar the backup metadata from the main backup file.
 
- 2. Untar the main backup file.
-
-        [user@restore ~]$ tar -i -xvf qubes-backup-2015-06-05T123456
+        [user@restore ~]$ tar -i -xvf qubes-backup-2015-06-05T123456 \
+            backup-header backup-header.hmac qubes.xml.000.enc
         backup-header
         backup-header.hmac
         qubes.xml.000.enc
```

Comment:
Is leaving these lines intentional?

Reply:
Oh, I see now. You're manually specifying the three files that should be extracted. In that case, shouldn't there be a `>` on the second line?

```
$ tar -i -xvf qubes-backup-2023-04-13T163220 \
> backup-header backup-header.hmac qubes.xml.000.enc
```

Reply:
> In that case, shouldn't there be a `>` on the second line (and likewise for other places where commands are broken up across multiple lines using `\`)?

That might cause some people to type in the `>` though

Reply:
It might, but it might also cause people to wonder why theirs has a `>` when the instructions don't and try to get rid of theirs (which they won't be able to). Is this a distro-specific convention or something common to all Linux environments? If the former, we can exclude it, but if the latter, we should include, IMHO.

Reply:
Seems to be pretty much universal. I'm not aware of any distro changing the secondary prompt string to something differnt.

One sharp edge might be that `>` is also the file redirection operator, so if `> foo bar baz` is entered as an independent line when the shell isn't in secondary prompt mode (e.g. forgot to type in the previous `\`) it would overwrite the file `foo`.

I can't tell anymore, multiline commands are weird. You decide :) Maybe after moving things back to #1279?

Reply:
What if we just didn't try to break up commands across multiple lines using `\` and instead left everything on a single line? If some commands seem too long for that, what if we directed the user to create a shell script file for them?

Reply:
That would feel the most natural to me. Might be tricky to get this to render ok on the website, if the long commands should have soft line wraps (not a mile-wide line to scroll through horizontally, but no hard line breaks either that would break copy and paste) yet still be distinguished from the output.

Reply:
> Might be tricky to get this to render ok on the website, if the long commands should have soft line wraps (not a mile-wide line to scroll through horizontally, but no hard line breaks either that would break copy and paste) yet still be distinguished from the output.

I'm afraid I don't have a solution for that part, except using in-line code snippets (i.e., wrap the command in single backticks) rather than putting it in a code block (i.e., having lines with triple backticks before and after).